js-exercises/minima/minima.js

```diff
@@ -0,0 +1,28 @@
+const isNumber = (num) => typeof num === 'number' && !Number.isNaN(num);
+
+function minima(numberOfElementsToReturn, array) {
```

Comment:
```suggestion
function minima(numerOfElements, array) {
```
In `numberOfElementsToReturn`, toReturn is generally implied since you are making a function which will return.

Reply:
Thank you for suggesting this.